pkg/cleaner/aws/aws.go

```diff
@@ -126,6 +135,40 @@ func (a *Cleaner) cleanBuckets() error {
 	return nil
 }
 
+func (a *Cleaner) cleanHostedZones() error {
+	var marker *string
+	for {
+		in := &route53.ListHostedZonesInput{
+			Marker: marker,
+		}
+
+		out, err := a.route53Client.ListHostedZones(in)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		for _, hz := range out.HostedZones {
+			if hz.Name == nil || hz.Id == nil {
+				continue
+			}
+
+			fmt.Printf("\n")
+			fmt.Printf("%#v\n", *hz.Id)
+			fmt.Printf("%#v\n", *hz.Name)
+			fmt.Printf("%#v\n", hz)
+			fmt.Printf("\n")
```

Comment:
This is just some listing of hosted zones. I do not know how to identify the ones we can just remove. I need feedback from @giantswarm/sig-infrastructure. <3